docs/cri/config.md

```diff
@@ -16,11 +16,18 @@ it is recommended to use the `systemd` driver on systemd-based hosts for complia
 [the "single-writer" rule](https://systemd.io/CGROUP_DELEGATION/) of cgroups.
 
 To configure containerd to use the `systemd` driver, set the following option in `/etc/containerd/config.toml`:
++ Before containerd 2.0
```

Comment:
I think it'd be nicer to swap the order so 2.0 instructions come first, and 1.x instructions come afterward.  We may also want to add a bit of formatting so "Before containerd 2.0" and "In containerd 2.0" stand out a bit; maybe a 4th-level heading or bold or something like that.

(Really, [tabbed panes](https://www.docsy.dev/docs/adding-content/shortcodes/#tabbed-panes) would be a really nice display, but not really something that is supported in GitHub markdown.)

Reply:
This just corresponds to:
https://github.com/containerd/containerd/blob/7a804489fdd528cc052071ce47d0217f3c6bcea9/docs/cri/registry.md?plain=1#L8-L17

The focus should be still put on v1.x, as nobody has begun to deploy v2.0 in production clusters yet.

Reply:
can see them both at the same time.. I read the desired outcome is going from 1.x -> to 2.x ..  and I agree 2.0 is not going to be looked at as production right away  maybe 2.0x or 2.1? vote is to swap order after 2.0 enters production realms..

Reply:
fixed